hostlists-builder/index.js

```diff
@@ -10,6 +10,7 @@ const HOSTLISTS_URL = 'https://adguardteam.github.io/HostlistsRegistry/assets';
 const CONFIGURATION_FILE = 'configuration.json';
```

Comment:
Since Mastodon logic is not related to hostlists and for better clarity I'd prefer to have it all in a separate file (`mastodon.js`) imported into `index.js`